packages/nuxt/src/index.types.ts

```diff
@@ -1 +1 @@
-export {};
+export * from './client';
```

Comment:
TODO: This will also have to export the types from the server, but we can do that in a follow up :)

Reply:
So far there is no server-side instrumentation. But when this gets added I will also add the types here 👍🏻